yamllintrc

```diff
@@ -0,0 +1,29 @@
+extends: default
+
+ignore:
+  - '**/hotsos/tests/unit/fake_data_root/**'
+  - '**/.tox/**'
+  - '**/examples/**'
+  - '**/.github/**'
+
+rules:
+  # https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.line_length
+  document-start:
+    present: false
+  # https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.line_length
+  line-length: disable
+  # https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.anchors
+  anchors:
+    forbid-undeclared-aliases: true
+    forbid-duplicated-anchors: true
+    forbid-unused-anchors: true
+  # https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.truthy
+  truthy:
+    allowed-values: ['true', 'false', 'on']
```

Comment:
on?

Reply:
I'm going to remove ` ignore: - '**/.github/**'` on the CI enablement patch, and 'on' is a keyword in GitHub workflow syntax so I'm excluding that.